brainstormforce/ultimate-addons-for-gutenberg
# Improvement: Added default hover border to editor

### Description
Added default hover border to the editor for block selection.

### Types of changes
non-breaking change 

### Test cases 1 -

- Deactivate Astra and check editor
- On block selection, a hover border appears or not.
- After selecting the block thin border appears or not just like Astra.

### Test cases 2 -

- Activate Astra and check editor
- On block selection, a hover border appears or not
- Astra border conflict with our plugin or not.

### Test cases 3 - 

- Drag and drop 3rd party plugins and core blocks also.
- Check hover hint from our plugin applied or not.



### Checklist:
